roles/openshift-applier/tasks/create-param-file-from-vars.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
It appears that this file is no longer in-use, so let's remove. (this is what confused me at first, thinking we didn't gp the `-p` route).

Reply:
removed.